frontend/src/static/css/style.scss

```diff
@@ -251,6 +251,38 @@ header {
     padding-right: 10px;
   }
 
+  #close-icon {
+    background-color: transparent;
+    border: 1px solid transparent;
+    bottom: 2%;
+    cursor: pointer;
+    display: none;
+    float: right;
+    margin: .5rem;
+    position: absolute;
+    right: -1rem;
+    vertical-align: middle;
+  }
+
+  #close-icon:after {
+    background-color: mui-color('grey', '600');
+    border-radius: 50%;
+    bottom: 5%;
+    color: white;
+    content: 'X';
+    cursor: pointer;
+    float: right;
+    font-size: .5rem;
+    font-weight: bold;
+    height: .5rem;
+    margin: auto;
+    padding: .1rem;
+    position: absolute;
+    right: 2%;
+    text-align: center;
+    width: .5rem;
+  }
+
```

Comment:
```
/* Prevents edge from adding a clear button in search input box */
.summary-picker__search::-ms-clear {
    display: none;
}
```

You'll need this style to prevent having two `X` on Edge.